azure-pipelines.yml

```diff
@@ -8,9 +8,13 @@ steps:
   inputs:
     versionSpec: '8.x'
   
-- script: |
-    npm install
-    npm test
+- task: Npm@1
+  inputs:
+    command: install
+
+- task: Npm@1
+  inputs:
+    command: test
```

Comment:
```yaml
displayName: 'npm test'
```

Reply:
Should be:

    - task: Npm@1
      displayName: 'npm test'
      inputs:
        command: custom
        customCommand: 'test'